jina/executors/indexers/vector.py

```diff
@@ -260,7 +292,8 @@ def _get_sorted_top_k(dist: 'np.array', top_k: int) -> Tuple['np.ndarray', 'np.n
 
         return idx, dist
 
-    def query(self, keys: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple[Optional['np.ndarray'], Optional['np.ndarray']]:
+    def query(self, queries: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple[
```

Comment:
Do not change the signature, it is inherited from another Base class, and we will need to change in every inherited class, so if we decide to do this, we will do it in a different PR

Reply:
Yes, reverted. I did add a overwrite inside the function. Is that OK? It was a bit confusing before, I thought. Especially since `keys` comes into play when figuring out the indices after masking